tests/Rector/ClassMethod/AddGenericReturnTypeToRelationsRector/Fixture/NewGenerics/belongs-to-many-with-no-pivot.php.inc

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace RectorLaravel\Tests\Rector\ClassMethod\AddGenericReturnTypeToRelationsRector\Fixture\NewGenerics;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class Website extends Model {}
```

Comment:
Could you add this to a Source folder withing the `AddGenericReturnTypeToRelationsRector` test folder. Sometimes adding classes into the test file leads to issues.

Reply:
Hi @peterfox, thank you for the feedback.

I'm not quite understanding the requirement here - the `Source` folder contains definitions for Old or New `Application` classes, and doesn't contain the fixture data. As far as I can tell, this does follow the existing structure, or have I missed something?

Thank you!

Reply:
@peterfox looking through one of your other PR's I think I get it - PR updated 👍